### PR TITLE
[Onepane] Adds Onepane Integration (ECOINT-3)

### PR DESCRIPTION
**Pull request by sajosam**
### What does this PR do?

This PR adds documentation for the Onepane integration, including an overview, setup instructions, uninstallation steps, and information about data collected

### Motivation

The motivation is to provide users with clear instructions on how to connect Datadog to Onepane, enabling them to leverage additional value from their existing Datadog investments. This integration allows users to correlate incidents with change data and infrastructure data, potentially improving root cause analysis and incident resolution

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [x] Feature or bugfix has tests
- [x] Git history is clean
- [x] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)
- [ ] If this PR includes a log pipeline, please add a description describing the remappers and processors. 

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

**Comment by drichards-87** [comment]
Created Jira card for Docs Team editorial review.

**Comment by eho1307** [comment]
Thanks for opening this PR! Dominic has reached out over email for additional information before we proceed with reviews.

File: onepane/README.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# Onepane
+
+## Overview
+
+Onepane is a powerful addition to your existing Datadog, Cloud, and DevOps tools, providing seamless out-of-the-box integrations. It adopts an agentless approach and features automatic asset discovery, allowing you to get value in a matter of hours, not days or weeks. Onepane comprehends your unique landscape, generating a dynamic service map of your operational environment by harnessing data from various systems. This map assists your teams in swiftly resolving incidents and streamlines the audit process. Opt for Onepane to leverage additional ROI on your existing investments. 
+
+- Show changes and infrastructure events alongside errors to pinpoint potential root causes.
+- Enable root cause investigation by correlating incidents with change data and infrastructure data.
+
+## Setup
```
Comment (emarsha94) [inline onepane/README.md:10]: OAuth must be kicked off from Datadog. Please include the exact steps a user will take within Datadog to set up the integration. For example, go to the Onepane integration on the integrations page, select install, click connect accounts, log in to your onepane account, .... etc

File: onepane/assets/dashboards/onepane_overview.json
Comment (emarsha94) [inline onepane/assets/dashboards/onepane_overview.json:1]: please update the dashboard using the dashboard template we provided via email

File: onepane/manifest.json [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+      "Category::AI/ML",
+      "Category::Cloud",
+      "Category::Automation",
+      "Category::AWS",
+      "Category::Azure",
+      "Category::Event Management",
+      "Category::Incidents",
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS"
```
Comment (emarsha94) [inline onepane/manifest.json:45]: This is missing required fields. Please make sure to include all required fields such as: 
Submitted Data Type:: (sending data in)
Queried Data Type:: (pulling data out) *If applicable
Offering:: 

See here https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags

File: onepane/manifest.json [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+      "source_type_id": 10449,
+      "auto_install": true
+    },
+    "dashboards": {
+      "Onepane Overview": "assets/dashboards/onepane_overview.json"
+    },
+    "oauth": "assets/oauth_clients.json"
+  },
+  "author": {
+    "support_email": "sajo@onepane.ai",
```
Comment (emarsha94) [inline onepane/manifest.json:72]: This should be an alias

File: onepane/README.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+- Access Integrations: Navigate to the "Integrations" section within the Onepane console.
+
+- Locate Datadog: Find the Datadog integration from the list.
+
+- Initiate Deletion: Click the three dots menu (ellipsis) in the top right corner of the Datadog integration tile. Select "Delete" from the options.
+
+- Confirm Deletion: A confirmation dialog might appear. If you're certain you want to proceed, click "Confirm" to remove the Datadog integration.
+
+By following these steps, you'll successfully disconnect your Datadog account from Onepane.
```
Comment (emarsha94) [inline onepane/README.md:59]: This should follow the oauth steps to uninstall the integration

File: onepane/manifest.json [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+      },
+      {
+        "media_type": "image",
+        "caption": "Datadog Integration",
+        "image_url": "images/integration.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Datadog Resource Mapping",
+        "image_url": "images/onepane_mapping.png"
```
Comment (emarsha94) [inline onepane/manifest.json:32]: Some of the images should include images of your datadog dashboard (once it's updated)

**Review by emarsha94 — CHANGES_REQUESTED** [review]

File: onepane/README.md (hunk 1 above)
Comment (emarsha94) [reply onepane/README.md:10]: The steps below do not walk through the OAuth flow. Please update the steps to walk through OAuth steps a user will take to install the integration as mentioned above. 

Users should not be creating api or app keys within their account as OAuth replaces the need for this.

File: onepane/README.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+- Authenticate with OAuth.
+
+    - Log in using your Datadog credentials.
+
+    - Upon successful authentication, Datadog will create an API key for you.
+
+- Provide a name for your Datadog connector.
+
+- Input the site URL and Application key obtained from the ![prerequisite steps][8].
+
```
Comment (emarsha94) [inline onepane/README.md:47]: OAuth must be intiated from Datadog. To make OAuth work for users across all Datadog regions, you will need to ensure that you’re making the correct API calls based on their region. **When the user kicks off authorization from the Datadog tile, a site parameter is sent on redirect from the onboarding_url. You will use this site parameter in your calls to the Authorize and Token endpoints. Please note that if a user kicks off authorization directly from your platform, this site parameter will not be sent, and the user will be prompted to select their site on the Datadog authorize page.**

File: onepane/README.md [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+- Access Integrations: Navigate to the "Integrations" section within the Onepane console.
+
+- Locate Datadog: Find the Datadog integration from the list.
+
+- Initiate Deletion: Click the three dots menu (ellipsis) in the top right corner of the Datadog integration tile. Select "Delete" from the options.
+
+- Confirm Deletion: A confirmation dialog might appear. If you're certain you want to proceed, click "Confirm" to remove the Datadog integration.
+
+When deleting the Datadog integration, the OAuth-generated API keys and other credentials will be soft-deleted and permanently deleted after 7 days.
```
Comment (emarsha94) [inline onepane/README.md:80]: Please update this to reflect Datadogs OAuth uninstallation steps

**Review by emarsha94 — CHANGES_REQUESTED** [review]

File: onepane/README.md [hunk 8]
```diff
@@ -0,0 +1,92 @@
+# Onepane
+
+## Overview
+
+Onepane is a GenAI tool that enhances incident resolution through automated Root Cause Analysis (RCA) by integrating seamlessly with monitoring tools like Datadog.
```
Comment (aliciascott) [inline onepane/README.md:5]:
```suggestion
Onepane is a GenAI tool that enhances incident resolution through automated root cause analysis (RCA) by integrating seamlessly with monitoring tools like Datadog.
```

File: onepane/README.md [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+By integrating Onepane with Datadog, customers gain rapid incident correlation with changes and infrastructure events, significantly reducing troubleshooting time and improving operational efficiency.
+The integration tracks incident data, infrastructure events, and change data (like code updates), providing valuable insights that help teams quickly identify root causes and prevent future issues.
```
Comment (aliciascott) [inline onepane/README.md:7]:
```suggestion
The integration tracks incident data, infrastructure events, and change data (such as code updates), providing valuable insights that help teams quickly identify root causes and prevent future issues.
```

File: onepane/README.md [hunk 10]
```diff
@@ -0,0 +1,92 @@
...
+
+
+## Setup
+
+**In Datadog:**
+
+- Go to **Integrations**, select the **Onepane** tile, and click **Install Integration**.
```
Comment (aliciascott) [inline onepane/README.md:14]:
```suggestion
- Navigate to **Integrations**, select the **Onepane** tile, click **Install Integration**.
```

File: onepane/README.md [hunk 11]
```diff
@@ -0,0 +1,92 @@
...
+
+- Click **Connect Accounts** to begin authorizing the integration. You will be redirected to [console.onepane.ai][2].
+
+**In Onepane:**
+
+- Log in if you're not already logged in with **Onepane**.
+
+- Please review the  the permissions and click on the 'Connect with datadog' button in the prompt.
```
Comment (aliciascott) [inline onepane/README.md:22]:
```suggestion
- Review the permissions and click the **Connect with Datadog** button in the prompt.
```

File: onepane/README.md [hunk 12]
```diff
@@ -0,0 +1,92 @@
...
+
+    ![Onepane_prompt][10]
+
+- You will be redirected back into Datadog to complete the authorization
```
Comment (aliciascott) [inline onepane/README.md:26]:
```suggestion
- You will be redirected back into Datadog to complete the authorization.
```

File: onepane/README.md [hunk 13]
```diff
@@ -0,0 +1,92 @@
...
+
+- Click **Create** to complete the integration. The Datadog connector will be deployed shortly.
+
+    ![Deploying][5]
+
+- View Host Resources: After successful deployment, you'll be able to see a list of host resources from your Datadog account within Onepane.
+
+    ![Host Resource][6]
+
+- Map Resources: Onepane allows you to map these Datadog resources to your existing Onepane resources for unified view of incidents and events across your infrastructure
```
Comment (aliciascott) [inline onepane/README.md:40]:
```suggestion
- Map Resources: Onepane allows you to map these Datadog resources to your existing Onepane resources for a unified view of incidents and events across your infrastructure.
```

File: onepane/README.md [hunk 14]
```diff
@@ -0,0 +1,92 @@
...
+
+    ![Onepane Mapping][7]
+
+With these steps complete, you'll have successfully integrated Datadog with Onepane
```
Comment (aliciascott) [inline onepane/README.md:44]:
```suggestion
With these steps complete, you'll have successfully integrated Datadog with Onepane.
```

File: onepane/manifest.json [hunk 15]
```diff
@@ -0,0 +1,101 @@
...
+  "manifest_version": "2.0.0",
+  "app_uuid": "68e9daf8-3439-49a0-8609-cdb91d72a036",
+  "app_id": "onepane",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Onepane is an GenAI tool that helps you faster incident resolution with automated Root Cause Analysis.",
```
Comment (aliciascott) [inline onepane/manifest.json:11]:
```suggestion
    "description": "Onepane is an GenAI tool that helps you faster incident resolution with automated root cause analysis.",
```

**Review by aliciascott — CHANGES_REQUESTED** [review]
Hi @sajosam in addition to the remaining comments from @emarsha94 , I added a few more suggestions, please let me know when resolved and I'll re-review, thanks!

File: onepane/assets/logos/onepane_dark.png
Comment (emarsha94) [inline onepane/assets/logos/onepane_dark.png:1]: Logo needs to be in svg format

File: onepane/manifest.json (hunk 5 above)
Comment (emarsha94) [reply onepane/manifest.json:32]: Please limit the images to 6 - thanks. Additionally, at least 1 image of the Datadog dashboard will need to be added as mentioned above.

**Review by emarsha94 — CHANGES_REQUESTED** [review]

**Review by aliciascott — APPROVED** [review]
approving the changes I requested, thanks for updating !

**Review by emarsha94 — APPROVED** [review]

File: .github/CODEOWNERS [hunk 16]
```diff
@@ -126,6 +126,7 @@
 /ns1/                                    @Zach-Johnson zjohnson@ns1.com @DataDog/ecosystems-review
 /nvml/                                   @cep21
 /octoprint/                              @gwaldo gwaldo@gmail.com
+/onepane/                                @sajosam sajo@onepane.ai @DataDog/ecosystems-review
```
Comment (bgoldberg122) [inline .github/CODEOWNERS:129]: Please also add the other CODEOWNER entries in the other section in this file. (see all `/ns1` entries for example)

**Review by bgoldberg122 — CHANGES_REQUESTED** [review]

**Review by iliakur — APPROVED** [review]

**Review by bgoldberg122 — APPROVED** [review]

**Pull request merged**